### PR TITLE
Add github actions to build binaries

### PR DESCRIPTION
**Pull request by bearice**
Provide prebuilt binaries to those who don't want to install the whole go compiler.

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.12
```
Comment (brancz) [inline .github/workflows/go.yml]: why not 1.16?

**Comment by brancz** [comment]
really nice idea! love go releaser!

File: .github/workflows/go.yml (hunk 1 above)
Comment (bearice) [reply .github/workflows/go.yml]: according to https://github.com/brancz/gojsontoyaml/blob/master/go.mod#L3

Comment (brancz) [reply .github/workflows/go.yml]: oh, do you mind bumping both of those? then I'm happy to merge and create a tag immediately! :)

**Comment by bearice** [comment]
go version bumped to 1.16

**Pull request merged**